Code/LauncherUnified/launcher_generator.cmake

```diff
@@ -257,6 +257,29 @@ foreach(project_name project_path IN ZIP_LISTS O3DE_PROJECTS_NAME LY_PROJECTS)
         # Associate the Unified Gem Variant with each projects UnfiedLauncher
         ly_set_gem_variant_to_load(TARGETS ${project_name}.UnifiedLauncher VARIANTS Unified)
     endif()
+
+    ################################################################################
+    # EditorWithServer
+    ################################################################################
+    if(PAL_TRAIT_BUILD_HOST_TOOLS)
+        if(PAL_TRAIT_BUILD_SERVER_SUPPORTED)
+            add_custom_target(${project_name}.EditorWithServer
+            )
+            ly_add_dependencies(${project_name}.EditorWithServer Editor)
+            ly_add_dependencies(${project_name}.EditorWithServer ${project_name}.ServerLauncher)
```

Comment:
Would you want the custom target to build the assets (TARGET `${project_name}.Assets`) as well or is that not needed?

Reply:
I don't think that's needed, since we're still running the Editor, which runs the AP, which builds the assets.

Reply:
Ok, I was wondering if there was a need to streamline the AssetProcessing, for the first time launch, but it looks like their isn't a need.